src/Psl/Collection/MutableAccessibleCollectionInterface.php

```diff
@@ -4,11 +4,11 @@
 
 namespace Psl\Collection;
 
+use ArrayAccess;
 use Closure;
 
 /**
- * The base interface implemented for a collection type that you are able set and remove its values.
- * keys.
+ * The base interface implemented for a collection type that you are able remove its values.
```

Comment:
This still supports setting, right? Not just removing?
```suggestion
 * The base interface implemented for a collection type whose values you are able to set and remove.
```

Reply:
indeed, the comment was modified after removing `set()` methods from the interface, but before adding `ArrayAccess`.